NuGet.Config

```diff
@@ -8,10 +8,19 @@
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
+    <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
   </packageSources>
   <packageSourceMapping>
     <clear />
-    <packageSource key = "dotnet-public">
+    <packageSource key="vs-impl">
+      <package pattern="Microsoft.VisualStudio.Markdown.*" />
+      <package pattern="Microsoft.VisualStudio.Shell.ViewManager" />
```

Comment:
I don't see this package being referred in `Directory.Packages.props`. This package is also not found in [`vs-impl`](https://dev.azure.com/azure-public/vside/_artifacts/feed/vs-impl) feed.